flake-registry.json

```diff
@@ -1,215 +1,5 @@
 {
   "flakes": [
-    {
-      "from": {
-        "id": "agda",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "agda",
-        "repo": "agda",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "arion",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "hercules-ci",
-        "repo": "arion",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "blender-bin",
-        "type": "indirect"
-      },
-      "to": {
-        "dir": "blender",
-        "owner": "edolstra",
-        "repo": "nix-warez",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "cachix",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "cachix",
-        "repo": "cachix",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "composable",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "ComposableFi",
-        "repo": "composable",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "disko",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "disko",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "dreampkgs",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "dreampkgs",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "dwarffs",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "edolstra",
-        "repo": "dwarffs",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "emacs-overlay",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "emacs-overlay",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "fenix",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "nix-community",
-        "repo": "fenix",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "flake-parts",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "hercules-ci",
-        "repo": "flake-parts",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "flake-utils",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "numtide",
-        "repo": "flake-utils",
-        "type": "github"
-      }
-    },
```

Comment:
I suspect that removing flake-parts and flake-utils might cause some update issues for people

Reply:
> And if they did, they will now see why this was a bad idea at all.

I get your point, but I don't think we should punish users for using an under-documented feature wrong. Most users aren't experts and are copy-pasting things from other places.

Reply:
Flakes are experimental, we are free to break related stuff. Lets just do it.